Gems/ROS2/Code/Include/ROS2/Sensor/ROS2SensorComponentBase.h

```diff
@@ -68,19 +69,50 @@ namespace ROS2
             return m_sensorConfiguration;
         }
 
+        void EnableSensor(bool sensorEnabled) override
```

Comment:
I suggest holding the same convention for all function in this header
Ex.
GetSensorConfiguration
SetSensorEnabled
SetPublishingEnabled
SetVisualizeEnabled
GetEffectiveFrequency
SetDesiredFrequency

Reply:
Agree,

Reply:
@patrykantosz could you refactor my PR following @MateuszWasilewski suggestion?

Reply:
@michalpelka Sure.